nixos/modules/services/mail/opensmtpd.nix

```diff
@@ -101,6 +102,18 @@ in {
       };
     };
 
+    security.wrappers.smtpctl = {
+      group = "smtpq";
+      setgid = true;
+      source = "${cfg.package}/bin/smtpctl";
+    };
+
+    services.mail.sendmailSetuidWrapper = mkIf cfg.setSendmail {
```

Comment:
Nit: could be written `mkIf cfg.setSendmail config.security.wrappers.smtpctl;` for DRY.

Reply:
Done.